Templates/Common-Backend-Scripts/pipelineBackend.config

```diff
@@ -34,15 +34,31 @@ logsDir="logs"
 ## End of Shared Configuration parameters    ########################################################
 #####################################################################################################
 
+
+#####################################################################################################
+## Shared Configuration parameters    ###############################################################
+#####################################################################################################
+
+# Setting to enable fetching build dependencies configured within the Application Descriptor
+# default: false
+fetchBuildDependencies=true
+
+#####################################################################################################
+## End of Shared Configuration parameters    ########################################################
+#####################################################################################################
+
 #####################################################################################################
 ## DBB-BUILD.sh parameters ##########################################################################
 #####################################################################################################
 
 # Path to main build script build.groovy.
 zAppBuild="$HOME/dbb-zappbuild"
+# DBEHM: MyCustomization for Path to build.groovy.
```

Comment:
Would it make sense to remove this line and the next one?

Reply:
This will be just for testing for the moment